lib/rubygems/specification.rb

```diff
@@ -2682,11 +2682,6 @@ def version=(version)
     @version = Gem::Version.create(version)
     return if @version.nil?
 
-    # skip to set required_ruby_version when pre-released rubygems.
-    # It caused to raise CircularDependencyError
```

Comment:
Need to update the comment on `def version=` above as well.

Reply:
Thanks for catching that!